src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -20,6 +21,14 @@ public static HotelDaoImpl getInstance() {
 
     @Override
     public List<Hotel> getAll() {
-        return hotelList;
+        return database;
+    }
+
+    public List<Hotel> getHotelsByName(String hotelName) {
+        return new ArrayList<Hotel>();
+    }
+
+    public List<Hotel> getHotelsByCity(String hotelCity) {
+        return new ArrayList<Hotel>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)